tests/lifecycle/driver_test.go

```diff
@@ -50,6 +53,9 @@ func TestServices(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
+	if err := crypto.InitializeGlobalCodec(noop.NewCodec()); err != nil {
```

Comment:
What's the purpose of this statement?

Reply:
It is to use `GetMapFromStruct` in mssql_cases_test.go. When `json.Marshal` hits our `SecureString` type, initialization for `crypto` is needed: https://github.com/Azure/open-service-broker-azure/blob/master/pkg/service/secure_string.go#L15.

Reply:
Thanks for the explanation.